src/EditorFeatures/Test/SignatureHelp/AbstractSignatureHelpProviderTests.cs

```diff
@@ -202,11 +209,36 @@ private void VerifyTriggerCharactersWorker(char[] expectedTriggerCharacters, cha
             }
         }
 
+        private class TestService : SignatureHelpService
```

Comment:
This seems weird.  Why not either:

1. Instantiate a SignatureHelpServiceWithProviders?
2. When would we want thsi service, instead of just using the real CSharp or VB SignatureHelpService?  

It feels like we shoudl use hte real service, but provide the specified provider alone.  I believe this is possible with completion, so we should do the same here.